package/Dockerfile

```diff
@@ -1,8 +1,21 @@
-FROM ghcr.io/oracle/oraclelinux:7-slim
+FROM ghcr.io/verrazzano/golang:v1.20.12 as build
+
 RUN yum update -y \
```

Comment:
This should be using "dnf", not yum.  I feel it would be easier and cleaner if you copied the template I pointed you to in the other project.

COPY . /go/src/github.com/backup-restore-operator/

RUN dnf install -y git && \
    cd /go/src/github.com/backup-restore-operator/scripts && \
    ./build && \
    ./test

Reply:
Thanks, made the changes.